### PR TITLE
Logging:deprecated warnings removed

### PR DESCRIPTION
**Pull request by abhinav-qlogic**

### CODE REVIEW

**Review by rahulKQL — APPROVED** [review]
LGTM

File: google-cloud-clients/google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingImpl.java [hunk 1]
```diff
@@ -629,7 +632,7 @@ public void onFailure(Throwable t) {
   static ListLogEntriesRequest listLogEntriesRequest(
       String projectId, Map<Option.OptionType, ?> options) {
     ListLogEntriesRequest.Builder builder = ListLogEntriesRequest.newBuilder();
-    builder.addProjectIds(projectId);
+    builder.addResourceNames(projectId);
```
Comment (rahulKQL) [inline google-cloud-clients/google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingImpl.java:635]: I just had a look at this Builder's [Javadoc](https://github.com/googleapis/google-cloud-java/blob/master/google-api-grpc/proto-google-cloud-logging-v2/src/main/java/com/google/logging/v2/ListLogEntriesRequest.java#L1002-L1006): 

> **these project identifiers are converted to resource name format** 

This line sounds like we have to convert the `projectId` before adding it in `resourceName`. Please confirm with someone who works/worked for this client.

cc:@ajaaym

File: google-cloud-clients/google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingImpl.java [hunk 2]
```diff
@@ -142,7 +143,8 @@ public Severity getFlushSeverity() {
           public O apply(I i) {
             return function.apply(i);
           }
-        });
+        },
+        MoreExecutors.directExecutor());
```
Comment (rahulKQL) [inline google-cloud-clients/google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingImpl.java]: Please use static import for `MoreExecutors.directExecutor()`.

File: google-cloud-clients/google-cloud-logging/src/test/java/com/google/cloud/logging/v2/testing/LocalLoggingImpl.java [hunk 3]
```diff
@@ -61,7 +61,7 @@ public void listLogEntries(
       ListLogEntriesRequest request, StreamObserver<ListLogEntriesResponse> responseObserver) {
     List<LogEntry> entries = new ArrayList<>();
 
-    for (ByteString proj : request.getProjectIdsList().asByteStringList()) {
+    for (ByteString proj : request.getResourceNamesList().asByteStringList()) {
```
Comment (rahulKQL) [inline google-cloud-clients/google-cloud-logging/src/test/java/com/google/cloud/logging/v2/testing/LocalLoggingImpl.java:64]: The former returns something like `{"[PROJECT_ID]" , ... }`  but 

what does getResourceNamesList() returns same value or with prefix type like `{"projects/[PROJECT_ID]", ...}`?

**Review by rahulKQL — CHANGES_REQUESTED** [review]

File: google-cloud-clients/google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingImpl.java (hunk 1 above)
Comment (abhinav-qlogic) [reply google-cloud-clients/google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingImpl.java:635]: ok i will conform and than update

File: google-cloud-clients/google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingImpl.java (hunk 2 above)
Comment (abhinav-qlogic) [reply google-cloud-clients/google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingImpl.java]: done

**Pull request closed without merging**